ranger/data/scope.sh

```diff
@@ -461,17 +462,15 @@ handle_mime() {
     esac
 }
 
-handle_fallback() {
-    echo '----- File Type Classification -----' && file --dereference --brief -- "${FILE_PATH}" && exit 5
-}
-
-
+# ENTRY POINT
+# =============================================================================
 MIMETYPE="$( file --dereference --brief --mime-type -- "${FILE_PATH}" )"
 if [[ "${PV_IMAGE_ENABLED}" == 'True' ]]; then
     handle_image "${MIMETYPE}"
 fi
 handle_extension
 handle_mime "${MIMETYPE}"
-handle_fallback
+# No handler for the current format or mimetype? Preview without color.
+cat -- "${FILE_PATH}"
```

Comment:
Are you sure this is a good idea? I have doubts about scrolling through large binary files: `cat` will take a ton of resources for actually nothing

Reply:
I think our priority should be addressing the case of use that is gonna affect 99% of users, which is file preview. But for the border case you mention ranger already has `preview_max_size`